modules/widgets/contact-info.php

```diff
@@ -90,49 +85,47 @@ function widget( $args, $instance ) {
 				$map_link = $this->build_map_link( $instance['address'] );
 
 				echo '<div class="confit-address"><a href="' . esc_url( $map_link ) . '" target="_blank">' . str_replace( "\n", "<br/>", esc_html( $instance['address'] ) ) . "</a></div>";
-
-
 			}
 
-
 			if ( $instance['phone'] != '' ) {
-
-				if( wp_is_mobile() ) {
-					echo '<div class="confit-phone"><a href="'. esc_url( 'tel:'. $instance['phone'] ) . '">' . esc_html( $instance['phone'] ) . "</a></div>";
-				} else {
+				if ( wp_is_mobile() ) {
+					echo '<div class="confit-phone"><a href="' . esc_url( 'tel:' . $instance['phone'] ) . '">' . esc_html( $instance['phone'] ) . "</a></div>";
+				}
+				else {
 					echo '<div class="confit-phone">' . esc_html( $instance['phone'] ) . '</div>';
 				}
-
 			}
 
-
 			if ( $instance['hours'] != '' ) {
 				echo '<div class="confit-hours">' . str_replace( "\n", "<br/>", esc_html( $instance['hours'] ) ) . "</div>";
 			}
 
-
-			echo $after_widget;
-
+			echo $args['after_widget'];
 		}
 
 
 		/**
 		 * Deals with the settings when they are saved by the admin. Here is
 		 * where any validation should be dealt with.
-		 **/
+		 *
+		 * @param array $new_instance New configuration values
+		 * @param array $old_instance Old configuration values
+		 *
+		 * @return array
+		 */
 		function update( $new_instance, $old_instance ) {
 			$update_lat_lon = false;
 			if ( $this->urlencode_address( $old_instance['address'] ) != $this->urlencode_address( $new_instance['address'] ) ) {
```

Comment:
I get a notice on this line when adding a new widget because `$old_instance` is an empty array. Can you please touch this up too?